### PR TITLE
Measures vignette

### PR DESCRIPTION
**Pull request by superp0tat0**
Add the measures vignette. Which consist of three major parts
* Why we need fairness measures
* Fairness Measures examples
    * Fairness measures which need only one metric
    * Fairness measures which need two or more metrics
    * How to customize Fairness measures from MeasureFairness Class
* How to compare fairness performance using Benchmarks

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -23,12 +23,14 @@ Imports:
     R6 (>= 2.4.1),
     data.table (>= 1.13.6),
     mlr3measures (>= 0.3.0),
-    mlr3misc
+    mlr3misc,
+    mlr3verse
```
Comment (pfistfl) [inline DESCRIPTION:27]: Do not import mlr3verse. Ever

File: vignettes/measures-vignette.Rmd [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+adult_train$col_roles$pta
+
+learner_obj = lrn("classif.rpart")
+learner_obj$train(adult_train)
+predictions = learner_obj$predict(adult_test)
+```
+
+## Measures only require one metric
+
+Users could use the following codesto create simple fairness measures which only requires one metric.
```
Comment (pfistfl) [inline vignettes/measures-vignette.Rmd:48]: code to..

File: vignettes/measures-vignette.Rmd [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+#Load the dataset, check the protected column has been set and make the predictions.
+adult_train = tsk("adult_train")
+adult_test = tsk("adult_test")
+adult_train$col_roles$pta
+
+learner_obj = lrn("classif.rpart")
+learner_obj$train(adult_train)
+predictions = learner_obj$predict(adult_test)
+```
+
```
Comment (pfistfl) [inline vignettes/measures-vignette.Rmd:45]: Structure: Start with motivation, then go into how to use the pre-defined measures  (via msr) and THEN show how to create your own measure.

File: vignettes/measures-vignette.Rmd [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+## Measures only require one metric
+
+Users could use the following codesto create simple fairness measures which only requires one metric.
+
+```{r}
+# False Positive Rate Bias
+me = MeasureFairness$new("groupwise_abs_diff", base_measure = msr("classif.fpr"))
```
Comment (pfistfl) [inline vignettes/measures-vignette.Rmd:52]: Just show 2-3 examples. Do not show all measures.

File: vignettes/measures-vignette.Rmd [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+# False negative error rate balance, also referred to as equal opportunity.
+# A classifier satisfies equal opportunity if the subjects in the protected and unprotected groups have equal FNR. We could use the following code to assess it:
+
+me = MeasureFairness$new(operation = "groupwise_quotient", base_measure = msr("classif.fnr"))
+predictions$score(me, task = adult_test) #Should close to 1
+
+# Equalized Odds
+# Users are expected to evaluate Equalized Odds by ratio or absolute difference. A classifier satisfies Equalized Odds if the subjects in the protected and unprotected groups have equal TPR and equal FPR. So we could use the following code to assess Equalized Odds:
+
+x = getFromNamespace("mlr_measures", ns = "mlr3")
```
Comment (pfistfl) [inline vignettes/measures-vignette.Rmd:140]: This should not be in the vignette. This is internal stuff

File: vignettes/measures-vignette.Rmd [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+  tasks = tsks(c("adult_train", "compas")),
+  learners = lrns(c("classif.ranger", "classif.rpart"),
+    predict_type = "prob", predict_sets = c("train", "test")),
+  resamplings = rsmps("cv", folds = 3)
+)
+
+bmr = benchmark(design)
+
+# Operations have been set to `groupwise_quotient()`
+measures = list( msr("fairness.tpr"), msr("fairness.npv"), msr("fairness.acc") )
```
Comment (pfistfl) [inline vignettes/measures-vignette.Rmd:161]: Also have a "classif.acc" for comparison

File: DESCRIPTION (hunk 1 above)
Comment (superp0tat0) [reply DESCRIPTION:27]: Removed the mlr3verse import

File: vignettes/measures-vignette.Rmd (hunk 2 above)
Comment (superp0tat0) [reply vignettes/measures-vignette.Rmd:48]: Sorry for the typo, corrected it.

File: vignettes/measures-vignette.Rmd (hunk 3 above)
Comment (superp0tat0) [reply vignettes/measures-vignette.Rmd:45]: Got it, reorganize the vignettes.

File: vignettes/measures-vignette.Rmd (hunk 4 above)
Comment (superp0tat0) [reply vignettes/measures-vignette.Rmd:52]: Change the examples amount to 3 (maximum) for each demonstration.

File: vignettes/measures-vignette.Rmd (hunk 6 above)
Comment (superp0tat0) [reply vignettes/measures-vignette.Rmd:161]: Done

**Comment by pfistfl** [comment]
Please try to solve the failing checks.

**Comment by pfistfl** [comment]
I am ok with this for now, Will do a pass later on. 
Can we merge @mllg ?

**Comment by mllg** [comment]
Yes, merge.

**Pull request merged**